python/app/services/user_service.py

```diff
@@ -1,10 +1,26 @@
 from app.db.session import SessionLocal
 from app.db.models.user import UserModel, UserType
+from app.graphql.types import User  # Import the GraphQL User type
+
+def convert_user_model_to_user(user: UserModel) -> User:
```

Comment:
Why are we doing this? 

This conversion is supposed to happen with strawberry_sqlalchemy_mapper by default

its anti pattern if we do this and also use strawberry_sqlalchemy_mapper

Reply:
changed this to normal fetching, faced issue fetching user details on the intial fetch so created this as an work around. now removed and tested working fine.